tests/ui/anonymous-higher-ranked-lifetime.stderr

```diff
@@ -26,7 +26,7 @@ LL |     f2(|_: (), _: ()| {});
    |     |
    |     expected due to this
    |
-   = note: expected closure signature `for<'a, 'b> fn(&'a (), &'b ()) -> _`
+   = note: expected closure signature `for<'b, 'a> fn(&'a (), &'b ()) -> _`
```

Comment:
This seems acceptable. Alternatively, we could sort this before displaying the diagnostics?